packages/shared/lib/utils/encryption.manager.ts

```diff
@@ -10,6 +10,8 @@ import type { RawDataRecordResult, DataRecord, DataRecordWithMetadata, RecordWra
 import db from '../db/database.js';
 import util from 'util';
 
+const { logger } = new Logger('Encryption.Manager');
+
 interface DataRecordJson {
     encryptedValue: string;
     [key: string]: any;
```

Comment:
I am tempted to also move `encryption.manager` into this new package. After all it was in utils before. The question is how to we deal with types?